librz/util/meson.build

```diff
@@ -112,7 +117,7 @@ else
   rz_util_sources += ['big.c']
 endif
 
-rz_util_includes = [platform_inc, include_directories('sdb/src', 'sdb')]
+rz_util_includes = [platform_inc, include_directories('sdb/src', 'sdb', '../include/rz_util', '.')]
```

Comment:
I do not think these additions are needed. Why did you add them?

Reply:
when I was testing (ninja -C build test), some error were there, showing
`FAILED: librz/util/librz_util.so.0.7.0.p/astr.c.o` and 
`/librz/include/rz_list.h:5:10: fatal error: ht_pp.h: No such file or directory`
    `5 | #include <ht_pp.h>`
    
so i thought adding `'.'` and `../include/rz_util` will specify the path to look for the header files (ht_*.h and set.h)